Libraries/Components/SafeAreaView/SafeAreaView.ios.js

```diff
@@ -13,13 +13,16 @@ const React = require('React');
 
 const requireNativeComponent = require('requireNativeComponent');
 
+import PropTypes from 'prop-types';
 import type {ViewProps} from 'ViewPropTypes';
 
 const RCTSafeAreaView = requireNativeComponent('RCTSafeAreaView');
 
-type Props = ViewProps & {
-  children: any,
-};
+type Props = $ReadOnly<{|
+  ...ViewProps,
+  children?: ?React.Node,
+  onInsetsChange?: ?Function,
```

Comment:
We could use a better flow type here, see 
https://github.com/clmntcrl/react-native/blob/d9568123d5f5dc491fd321e2ed463bffb13868ec/Libraries/Components/Switch/Switch.js#L66 for example.